core/modules/wsched/BlendScheduler.cc

```diff
@@ -137,6 +137,134 @@ void BlendScheduler::_sortScanSchedulers() {
 }
 
 
+void BlendScheduler::queCmd(util::Command::Ptr const& cmd) {
+    std::vector<util::Command::Ptr> vect;
+    vect.push_back(cmd);
+    queCmd(vect);
+}
+
+
+/// &&& Change this to accept a vector of util::Command::Ptr const&
+void BlendScheduler::queCmd(std::vector<util::Command::Ptr> const& cmds) {
+    // Do the book keeping and determine which queue the tasks in 'cmds'
+    // should be added to. All tasks in 'cmds' must belong to the same
+    // user query and will go to the same scheduler.
+    LOGS(_log, LOG_LVL_WARN, "&&& BlendScheduler::queCmd a");
+    bool first = true;
+    SchedulerBase::Ptr s = nullptr;
```

Comment:
If this is the shared pointer then you don't need to explicitly initialize it with `nullptr`. This is done by the default c-tor of class `std::shared_ptr<T>`.
My second comment is that that name `s` is pretty much meaningless. Could some reasonable name be given to it to reflect a role of the variable?